linera-views-derive/src/lib.rs

```diff
@@ -128,7 +128,7 @@ fn generate_view_code(input: ItemStruct, root: bool) -> TokenStream2 {
     let increment_counter = if root {
         quote! {
             linera_views::increment_counter(
-                linera_views::LOAD_VIEW_COUNTER,
+                &linera_views::LOAD_VIEW_COUNTER,
                 stringify!(#struct_name),
                 &context.base_key(),
             );
```

Comment:
This call here (and only this call) hangs, which causes things to timeout and the test to fail. Not sure why though, I thought it might be because it's called from inside the derive macro code and the `Lazy` wouldn't be initialized properly, but I tried forcing initialization and still had the issue.